src/Razor/src/Microsoft.NET.Sdk.Razor/Microsoft.NET.Sdk.Razor.csproj

```diff
@@ -36,13 +36,15 @@
     <Compile Include="..\Microsoft.AspNetCore.Razor.Tools\Client.cs">
       <Link>Shared\Client.cs</Link>
     </Compile>
+
+    <Content Include="_._" CopyToOutputDirectory="PreserveNewest" />
```

Comment:
FYI @rynowak \ @NTaylorMullen, this is working around a fairly new warning in NuGet:

https://github.com/NuGet/NuGet.Client/commit/ab141634c0e98655153d989a967655710bb6ee14

```
D:\work\AspNetCore-Tooling\.dotnet\sdk\5.0.100-alpha1-013788\Sdks\NuGet.Build.Tasks.Pack\buildCrossTargeting\NuGet.Build.Tasks.Pack.targets(198,5): warning NU5127: This package does not contain a lib/ or ref/ folder, and will therefore be treated as compatible for all frameworks. Since framework specific files were found under the build/ directory for netstandard2.0, consider creating the following empty files to correctly narrow the compatibility of the package: [D:\work\AspNetCore-Tooling\src\Razor\src\Microsoft.NET.Sdk.Razor\Microsoft.NET.Sdk.Razor.csproj]
```